apps/src/weblab/brambleHost.js

```diff
@@ -0,0 +1,195 @@
+/* global requirejs */
+
+/**
+ * JS to communicate between Bramble and Code Studio
+ */
+
+(function () {
+    window.requirejs.config({
+        baseUrl: '/blockly/js/bramble/'
+    });
+
+    // the main Bramble object -- used to access file system
+    var bramble_ = null;
+    // the Bramble proxy object -- used to access methods on the Bramble UI frame
+    var brambleProxy_ = null;
+    // interface to Web Lab host in Code Studio
+    var webLab_ = null;
+
+    // Project root in file system
+    var projectRoot = "/codedotorg/weblab";
+
+    function putFilesInBramble(Bramble, sources, callback) {
+        var fs = Bramble.getFileSystem();
+        var sh = new fs.Shell();
+        var Path = Bramble.Filer.Path;
+
+        sh.rm(projectRoot, {recursive: true}, function (err) {
```

Comment:
could make a lot of these callbacks arrow funcs now if you wanted, now that we're going through our JS pipeline